_layouts/home.html

```diff
@@ -10,6 +10,10 @@
 
   <body class="{% if page.type == "tag"%}tag-{% endif %}page-{{ page.title | slugify }}
   {% if page.header_border == true or layout.header_border == true %}header_border{% endif %} {% if page.path contains '_posts' %}single-post{% endif %}">
+    <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KHXPF5R"
```

Comment:
Why is it in two places?

Reply:
Because of the instructions saying the 1st part to be in the head section while the 2nd part to be in the body section: 
![image](https://user-images.githubusercontent.com/13788498/80544367-03b6c280-8a05-11ea-83bf-1ba471c9cbf7.png)
![image](https://user-images.githubusercontent.com/13788498/80544395-14673880-8a05-11ea-8113-7d88230add53.png)

Reply:
Any idea which place it should be? Just because not every page uses the same layout